cross/syncthing/Makefile

```diff
@@ -37,21 +36,14 @@ ifeq ($(SYNCTHING_ARCH),)
 $(error Unsupported ARCH $(ARCH))
 endif
 
-ifeq ($(findstring $(SYNCTHING_ARCH),amd64),$(SYNCTHING_ARCH))
-SYNCTHING_BIN = $(WORK_DIR)/$(PKG_DIR)/bin/syncthing
-endif
-
-ifeq ($(findstring $(SYNCTHING_ARCH),386),$(SYNCTHING_ARCH))
-SYNCTHING_BIN = $(WORK_DIR)/$(PKG_DIR)/bin/linux_386/syncthing
-endif
-
-ifeq ($(findstring $(SYNCTHING_ARCH),arm),$(SYNCTHING_ARCH))
-SYNCTHING_BIN = $(WORK_DIR)/$(PKG_DIR)/bin/linux_arm/syncthing
-endif
+SYNCTHING_DIR = $(WORK_DIR)/src/github.com/$(PKG_NAME)/$(PKG_NAME)
+EXTRACT_PATH = $(WORK_DIR)/src/github.com/$(PKG_NAME)/
```

Comment:
Nit: Let's remove the trailing slash (https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.extract.mk#L9)